### PR TITLE
Improve Google Sheets Sync feature

### PR DESCRIPTION
**Pull request by piyarathnalakmali**
**Note: DefectDojo is now on Python3 and Django 2.2.1 Please submit your pull requests to the 'dev' branch as the 'legacy-python2.7' branch is only for bug fixes. Any new features submitted to the legacy branch will be ignored and closed.**

When submitting a pull request, please make sure you have completed the following checklist:

- [x] Your code is flake8 compliant 
- [x] Your code is python 3.5 compliant

I have done two ain modifications to the Googe Sheets Sync feature 

- If a finding is a duplicate finding, display notes of the original finding in the Google Sheet. 
- Allowed the system admin to provide the Gmail address used to create the service account. Gave editing permission to that Gmail address for all protected ranges.

### CODE REVIEW

**Comment by valentijnscholten** [comment]
There are new migrations in `dev`, could you please update this PR accordingly?

**Comment by madchap** [comment]
Hi @piyarathnalakmali 

Could you please rebase?

Also, I opened a couple issues when trying the current code: https://github.com/DefectDojo/django-DefectDojo/issues/1993 and https://github.com/DefectDojo/django-DefectDojo/issues/1991, it would be superb if you could take a look :-)

**Comment by madchap** [comment]
Integration tests crashed, closing and re-opening.

File: dojo/forms.py [hunk 1]
```diff
@@ -1939,6 +1939,10 @@ class GoogleSheetFieldsForm(forms.Form):
         required=True,
         label="Google Drive folder ID",
         help_text="Extract the Drive folder ID from the URL and provide it here")
+    gmail_address = forms.EmailField(
```
Comment (madchap) [inline dojo/forms.py:1942]: I don't think we should specifically call out gmail. I use the feature and use GSuite, as such, I don't use a @gmail.com address.

File: dojo/google_sheet/views.py [hunk 2]
```diff
@@ -39,6 +39,7 @@ def configure_google_sheets(request):
             else:
                 initial['Protect ' + field.name] = True
         initial['drive_folder_ID'] = system_settings.drive_folder_ID
+        initial['gamil_address'] = system_settings.email_address
```
Comment (ayomawdb) [inline dojo/google_sheet/views.py:42]: Shall we update the label here also?
```suggestion
        initial['email_address'] = system_settings.email_address
```

Comment (madchap) [reply dojo/google_sheet/views.py]: Thanks for the catch @ayomawdb  :)

**Comment by madchap** [comment]
Re-triggering Travis tests.

**Comment by mtesauro** [comment]
FWIW, looks good to me assuming @madchap requested changes happen.

**Comment by fabform** [comment]
This is handy to find google sheets id
[find google sheet id](https://fabform.io/a/how-to-find-google-sheet-id)

**Pull request merged**